pyproject.toml

```diff
@@ -20,16 +20,16 @@ Jinja2 = "==3.1.1"
 MarkupSafe = "==2.1.1"
 packaging = "==21.3"
 prompt-toolkit = "==3.0.28"
-python = "^3.7"
-PyYAML = "==6.0"
+python = "==3.9.10"
+PyYAML = "==6.0.1"
```

Comment:
I was seeing [test failures](https://app.circleci.com/pipelines/github/artsy/hokusai/843/workflows/1d1640d0-1db0-49ab-83be-34f3dd3c5869/jobs/3612?invite=true#step-105-214) resolving `pyYAML = "==6.0"` that were fixed with this `patch` version bump.